lib/offline/db_engine.js

```diff
@@ -229,7 +229,7 @@ shaka.offline.DBEngine.prototype.removeSegments =
     function(keys, onKeyRemoved) {
   return this.remove_(
       shaka.offline.DBEngine.Store.SEGMENT,
-      keys,
+      keys.filter(function(key) { return key !== null; }),
```

Comment:
We typically prefer '==/!=' to '!==/===' unless there is a good reason to use the latter.

(Here and below)

Reply:
@ismena, changed. But why do you prefer == over ===?